db/migrate/20210806041206_users.rb

```diff
@@ -1,13 +0,0 @@
-class Users < ActiveRecord::Migration[6.1]
```

Comment:
This won't be possible in a production application.

You can't just run twice a migration for creating a table that already exists.

And well, quoting the Rails Guides:

Migrations are a convenient way for you to alter your database in a structured and organized manner. You could edit fragments of SQL by hand but you would then be responsible for telling other developers that they need to go and run them. You’d also have to keep track of which changes need to be run against the production machines next time you deploy.

We keep every migration since each of them has a version number, which is stamped in the database that tell us in which order to run them.

However, I'd recommend instead of deleting this migration file, to modify the one generated by devise...

If you see the schema-changes, there are a couple of new columns. Pls, modify the new migration to instead fo "creating the users table" to add the missing columns. Also remove some of the columns that won't be needed anymore, like password. Devise stamps the password encrypted by default. I'd recommend avoid future confusion and delete it.

Also, If you need help for this, feel free to DM and we can put some time in our cal to pair on this

I'd be super happy to pair with you if needed 😄

Reply:
Good catch Eduardo!

@SampabloC in this case that you already have the `users` table, you can modify it by following this Devise [link](https://github.com/heartcombo/devise/wiki/How-To:-Change-an-already-existing-table-to-add-devise-required-columns)

Please let us know if you have any questions 😀